src/renderer/views/Watch/Watch.scss

```
@@ -650,6 +655,25 @@
       box-shadow: none;
     }
 
+    .shortsAuxPanelTarget :deep(.watchVideoTranscript) {
```

Stylelint (17.14.1)

[error] Unknown pseudo-class selector ":deep" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)